run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -eu
+
+#set the max memory
+BYTES_PER_MEG=$((1024*1024))
+BYTES_PER_GIG=$((1024*${BYTES_PER_MEG}))
+
+DEFAULT_MIN=$((64 * $BYTES_PER_MEG)) #This is a guess
+regex='^([[:digit:]]+)([GgMm])?i?$'
+num_regex='^[[:digit:]]+'
+unit_regex='[GgMm]'
+
+export NODE_OPTIONS=""
+
+if [ 1 -eq $(echo "${OCP_AUTH_PROXY_MEMORY_LIMIT:-}" | grep -Ec $regex) ] ; then
```

Comment:
Is all this grep business really preferable to using bash regex matching [like we do for elasticsearch](https://github.com/openshift/origin-aggregated-logging/blob/e2314101a1c53226a4d2563efa7f942807f31537/elasticsearch/run.sh#L31)?